.github/workflows/development-environment.yml

```diff
@@ -31,14 +30,29 @@ jobs:
       - name: Checkout sentry
         uses: actions/checkout@v2
 
+      # The next few steps are to set up the cache quickly
+      - name: Set environment variables & others
+        id: brew-info
+        run: |
+          echo "::set-output name=brew-cache::$(brew --cache)"
+
+      - name: Cache (brew)
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.brew-info.outputs.brew-cache }}
+          key: devenv-${{ runner.os }}-brew-${{ hashFiles('Brewfile') }}
+          restore-keys: devenv-${{ runner.os }}-brew
+
       - name: Install prerequisites
         # Xcode CLI & brew are already installed, thus, no need to call xcode-select install
         # Sometimes, brew needs to be updated before brew bundle would work
         # After installing Docker (via homebrew) we need to make sure that it is properly initialized on Mac
         run: |
-          brew update -q && brew bundle -q
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask docker || rm -rf $(brew --cache) && brew install --cask docker
           # This code is mentioned in our dev docs. Only remove if you adjust the docs as well
           SENTRY_NO_VENV_CHECK=1 ./scripts/do.sh init-docker
+          # We remove the cache if we fail since signature mistmatches can happen
+          HOMEBREW_NO_AUTO_UPDATE=1 brew bundle --no-upgrade || rm -rf $(brew --cache) && brew bundle
```

Comment:
I don't know why but we I believe the Github runners got a bad Docker image in their cache and it needs to be removed.